pkg/cache/kubeConfig.go

```diff
@@ -22,19 +22,19 @@ var kubeConfigCache = ttlcache.New[string, string](
 // If it is expired, it will get the kubeconfig from the secret and set it in the cache.
 func GetKubeConfigFromCache(logger *zap.SugaredLogger, clientSet *kubernetes.Clientset, runtimeID string) (string, error) {
 	kubeConfigCache.DeleteExpired()
-	kubeConfigFromCache := kubeConfigCache.Get(runtimeID)
 
-	if kubeConfigFromCache.Value() == "" || kubeConfigFromCache.IsExpired() {
-		kubeConfigCache.Delete(runtimeID)
-		kubeConfig, err := getKubeConfigFromSecret(logger, clientSet, runtimeID)
-		if err == nil {
-			kubeConfigCache.Set(runtimeID, kubeConfig, 5*time.Minute)
-		}
-		return kubeConfig, err
+	if kubeConfigCache.Has(runtimeID) {
```

Comment:
Looks good, but now there is the scenario missing when the kubeconfig is expired, please add this again